moveit_ros/benchmarks/CMakeLists.txt

```diff
@@ -1,39 +1,54 @@
 cmake_minimum_required(VERSION 2.8.12)
 project(moveit_ros_benchmarks)
 
+set(MOVEIT_LIB_NAME moveit_ros_benchmarks)
+
 add_compile_options(-std=c++11)
 
 if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
 endif()
 
-find_package(Boost REQUIRED COMPONENTS date_time filesystem program_options system thread)
+find_package(Boost REQUIRED filesystem)
+
+# In ROS Hydro, MoveIt links against version 2 of Boost Filesystem
```

Comment:
This hack for Hydro/Boost is definitely not necessary anymore, can you remove it and the compiler directives?